src/app/components/Tooltip/index.tsx

```diff
@@ -6,16 +6,21 @@ import styled from 'styled-components';
 import Popover, { PopoverProps, PopperWithoutArrowAndBorder } from '../Popover';
 
 const TooltipContainer = styled.div<{ wide?: boolean; small?: boolean }>`
-  @media (max-width: 650px) {
-    ${props => props.small && ' width: 156px; padding: 11px;'}
-  }
-
   width: ${props => (props.wide ? '300px' : '244px')};
   padding: 10px 0.9375rem;
   line-height: 150%;
   font-weight: 400;
   font-size: 14px;
+  margin: 0 10px;
```

Comment:
Let's remove this margin, as it makes the text look too centered.

If this was to fix the spacing issue I mentioned, you may have misunderstood what I was referring to. As soon as you reduce the screen size, the tooltips hug right up against the edge:
![Screen Shot 2021-08-19 at 12 38 08 PM](https://user-images.githubusercontent.com/63842577/129989707-70ec4d2b-daeb-4cd4-a386-68a63f1a566d.png)
![Screen Shot 2021-08-19 at 12 37 55 PM](https://user-images.githubusercontent.com/63842577/129989704-7699696c-9532-4864-9ca3-b196313e6d7b.png)

Reply:
Tooltip size set to 260px. 
Can't do much about the screen hugging, but it never goes off the screen.